client/vue.config.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
   chainWebpack: (config) => {
     config.output.strictModuleExceptionHandling(true);
     config.resolve.symlinks(false);
-    config.resolve.alias.set('app', path.resolve(__dirname, 'app'));
+    config.resolve.alias.set('@', path.resolve(__dirname, 'viame-web-common'));
+    config.resolve.alias.set('viame-web-common', path.resolve(__dirname, 'viame-web-common'));
     config.resolve.alias.set('vue-media-annotator', path.resolve(__dirname, 'src'));
```

Comment:
Can mark as resolved, would it make  more sense for consistent language and vocabulary if we renamed `./src` to `./vue-media-annotator`?  That way everything in it is already referred to by being the media-annotator.

Reply:
Let's do this later.  

1) It's common convention for the distributable package to live in src/
2) This PR has already been mega-disruptive, and this change isn't necessary for the purpose of the PR.